.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
 npm-debug.log
+obj/
```

Comment:
is it a leftover from 'dotnet restore'

Reply:
it keeps auto building all these , so may as well ignore the directory